docs/operator-manual/config-management-plugins.md

```diff
@@ -178,7 +178,7 @@ entrypoint. You can use either off-the-shelf or custom-built plugin image as sid
 containers:
 - name: my-plugin
   command: [/var/run/argocd/argocd-cmp-server] # Entrypoint should be Argo CD lightweight CMP server i.e. argocd-cmp-server
-  image: busybox # This can be off-the-shelf or custom-built image
+  image: registry.access.redhat.com/ubi8 # This can be off-the-shelf or custom-built image
```

Comment:
```suggestion
  image: ubuntu # This can be off-the-shelf or custom-built image
```

Will this one work? I agree with @wanghong230 that an example should be simple and I don't see a reason to add something from the RedHat registry.

Reply:
Yeah, ubuntu would be my preference as well